src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -1225,7 +1225,18 @@ namespace
 
             QVBoxLayout* layout = new QVBoxLayout();
             container()->setLayout(layout);
-
+            #ifdef APPLESEED_WITH_EMBREE
```

Comment:
It's customary to omit indentation for preprocessor statements, and we follow that tradition (fix everywhere).